analyses/cms-open-data-ttbar/utils.py

```diff
@@ -24,10 +52,95 @@ def update_to(b=1, bsize=1, tsize=None):
 
     return update_to
 
-def cache_files(file_paths: list, cache_dir: str, remote_prefix: str):
+
+def _cache_files(file_paths: list, cache_dir: str, remote_prefix: str):
     for url in file_paths:
-        out_path = Path(cache_dir) / url.removeprefix(remote_prefix).lstrip('/')
+        out_path = Path(cache_dir) / url.removeprefix(remote_prefix).lstrip("/")
         out_path.parent.mkdir(parents=True, exist_ok=True)
         if not out_path.exists():
-            with tqdm(unit='B', unit_scale=True, unit_divisor=1024, miniters=1, desc=out_path.name) as t:
+            with tqdm(
+                unit="B", unit_scale=True, unit_divisor=1024, miniters=1, desc=out_path.name
+            ) as t:
                 urlretrieve(url, out_path.absolute(), reporthook=_tqdm_urlretrieve_hook(t))
```

Comment:
This means we can't cache files when the input paths use an xrootd prefix. It's not for this PR, I'll open a github issue